websockets/1.0/modules/ROOT/pages/websockets-connector-examples.adoc

```diff
@@ -239,14 +234,74 @@ The received `QuoteSnapshots` should be transformed to CSV format and appended t
 			</when>
 		</choice>
 	</flow>
+
+</mule>
+----
+
+== Quote Aggregator App
+
+The following Mule app connects to the feed endpoint of the previous Quote Producer App and receives all of the quotes. Then, the Quote Aggregator App splits the quotes by ticker and places them in time-based aggregators that occur every 5 seconds.
```

Comment:
Global, IMHO, should be "Quote Aggregator app" unless App is part of the example name in the code.